exercicios/para-casa/dataset_alunas.py

```diff
@@ -59,4 +59,129 @@
         "Presença": [True, True, True, False, True],
         "Participação": 7.5
     }
-}
\ No newline at end of file
+}
+
+
+
+print ("-------------------------hello world!!---------------------------------")
+print ("-------------------Esse é o sistema reprograma-------------------------")
+print ("-----------------------seja bem vinde ;) ------------------------------")
+print ("----------Para ter acesso a lista de alunas (digite 1)-----------------")
+print ("----------------------Para inserir aluna nova (digite 2)--------------")
+print ("-------------------Para consultar presença (digite 3)-----------------")
+print ("----------------Para consultar notas  (digite 4)------------------------")
+print ("----------------Para calcular média (digite 5)------------------------")
+print ("-----------------Para sair do sistema (digite 6)----------------------")
+
+ 
+
+ 
+
+     
+    
+def inclusao_novas_alunas ():
+    while True:
```

Comment:
Não necessita de while True no início das funções, a não ser que você realmente queira que ele tenha mais de uma iteração. Como você escreveu um break no fim, acredito que não seja esse o caso. Um exemplo de como poderiamos incluir um while nesse caso seria, manter o usuário nesta função enquanto não conseguissemos incluir a aluna. Revise a aula de loops, ou a aula de sábado para olhar este ponto.